trio/_highlevel_open_tcp_listeners.py

```diff
@@ -6,7 +6,7 @@
 from math import inf
 
 import trio
-from trio.lowlevel import TaskStatus
+from trio import TaskStatus
```

Comment:
Note to self: this isn't actually a breaking change (I got concerned for a bit, oops).